Jellyfin.Api/Controllers/LiveTvController.cs

```diff
@@ -1071,7 +1071,7 @@ public async Task<ActionResult> GetSchedulesDirectCountries()
         {
             var client = _httpClientFactory.CreateClient();
             // https://json.schedulesdirect.org/20141201/available/countries
-            using var response = await client.GetAsync("https://json.schedulesdirect.org/20141201/available/countries")
+            var response = await client.GetAsync("https://json.schedulesdirect.org/20141201/available/countries")
```

Comment:
Why?

Reply:
having `using` there disposes the stream before the aspnet framework can return it, resulting in `Cannot access a closed Stream.`

Reply:
Maybe add a comment?